src/index.html

```diff
@@ -1,17 +1,212 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+     content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="form">
+
+        <legend>Personal information</legend>
```

Comment:
You don't need a line break here. Don't add line breaks between parent and child tags
```suggestion
      <fieldset class="form">
        <legend>Personal information</legend>
```

Reply:
Fix this in all places

Reply:
Okey, thanks, i'll fix it.